code/modules/jobs/job_types/peasants/jester.dm

```diff
@@ -50,7 +50,7 @@
 	H.adjust_skillrank(/datum/skill/misc/athletics, pick(4,4,4,4,5), TRUE)
 	H.adjust_skillrank(/datum/skill/misc/music, pick(1,2,3,4,5,6), TRUE)
 	H.adjust_skillrank(/datum/skill/craft/cooking, pick(1,2,3,4,5,6), TRUE)
-	H.adjust_skillrank(/datum/skill/combat/firearms, pick(1,2,3,4,5,6), TRUE)
+	H.adjust_skillrank(/datum/skill/combat/firearms, pick(0,0,0,0,0,6), TRUE)//I think making it a 1/6 chance is even funnier
```

Comment:
i agree with you but atleast make it a 1,1,1,1,1,6 because with 0 firearms skills you can't even reload a puffer gun in the 1% chance that the jester manages to get one